### DISCUSSION TITLE
Just Thanks for continuing this Project

### DISCUSSION THREAD

**Discussion by lux73** [ReimuHakurei/phpPgAdmin#6] [General]
Just wanna say Thanks for continuing this Project!

Really appreciated your Work :-)

**Comment by leokasion** [comment]
I wanted to write the same, thanks for continuing this important project so far from Gensokyo, it really helped me because pgadmin4 it's a pain to install it on Nginx because it has to run as a webapp using Gunicorn under the proper Python venv...
